internal/utils/symlinks.go

```diff
@@ -27,12 +27,9 @@ func makeLink(from string, to string, name string) string { // {{{
 	} else if PathExists(to + name) {
 		fmt.Println("Symlink already exists, removing and relinking...")
 		err := os.RemoveAll(to + name)
-
 		/* This remove a directory and replaces it with the new synlink in case it already exists.
 		The reason to use RemoveAll was that Remove cannot delete a directory if it is not empty.
-		On Unix, RemoveAll uses `rm -rf Dir/`*/
-
-		if err != nil {
+		On Unix, RemoveAll uses `rm -rf Dir/`*/ if err != nil {
```

Comment:
what happened here with the formatter? :monocle_face:

Reply:
`gofumpt` seems to like doing this with multi-line comments.
I can change this back, shouldn't affect the code either way.

Reply:
just jumped out to me, seems like it'd be easier to miss.